tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchTests.java

```diff
@@ -0,0 +1,259 @@
+package org.alfresco.elasticsearch;
```

Comment:
I guess this class should have an Enterprise License header

Reply:
Other tests don't have it.